.github/workflows/build-bigsur-bottles.yml

```diff
@@ -0,0 +1,16 @@
+name: Build BigSur bottles
+on:
+  push:
+    branches:
+      - PruStephan/Remove_unnecessary_tezos_client_deps
+jobs:
+  build-bottles:
+    name: Install and test binaries
+    runs-on: macos-11
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Build bottles and print dependencies
```

Comment:
Hmm, do I understand correctly that the only purpose of this script is to list deps?
If so, is it possible to assert that all deps are up-to-date, i.e. there are no excessive deps

Reply:
Do you plan to keep this check in the CI pipeline or was it used only for listing deps to avoid building everything locally?
If you're planning to keep it, then something has to be done with the fact that it takes **a lot** of time to complete and it's not viable to run it on every push to every branch

Reply:
I was not planning to leave this script. Installing all the bottles using gh actions takes a lot of time and space (And I am not sure which one prevents it from finishing).

Reply:
> Hmm, do I understand correctly that the only purpose of this script is to list deps?
If so, is it possible to assert that all deps are up-to-date, i.e. there are no excessive deps

If I understand correctly how otool works (i.e. It does not list deps that are listed in .rb file, but instead list dependencies that are actually used by binary) yes, it simply lists deps for all files. To do that requires all the binaries to be built, and thus is not very useful. See my comment above